### ISSUE TITLE
adb error:failed to copy 'libs/x86/minicap' to 'C:/Program Files/Git/data/local/tmp/minica-devel'

### ISSUE THREAD

**Issue by sreenivash** [openstf/minicap#95]
**What is the issue or idea you have?**
- While running 'run.sh -P 768x1280@720x1280/0' getting 
adb error:failed to copy 'libs/x86/minicap' to 'C:/Program Files/Git/data/local/tmp/minica-devel'

**Have you tried STF?**

**Comment by sorccu** [comment]
Windows is not supported, I have no time for that. Please look at the script and try to do what it does manually.

**Comment by sreenivash** [comment]
Thanks sorccu

**Comment by plantyoon** [comment]
@tsreenivash-pk 
It's late, I had the same problem, but I have confirmed that it works well in Windows by modifying line 46 of run.sh. I modified the libs folder address to an absolute path. It is my 46th line.
```
adb push /E/Projects/STUDY/minicap/libs/$abi/$bin $dir
```


And when you run run.sh, you will get a `Permission denied` error like a #37.
Add the line `adb shell chmod 777 /data/local/tmp/minicap-devel/minicap` to the 55th line `# Run!` comment, as shown in the final answer to Issue #37.


![window-test](https://i.imgur.com/G5EltXD.jpg)
